lua/lean/init.lua

```diff
@@ -103,10 +103,16 @@ function lean.current_search_paths()
     paths = require'lean.lean3'.__current_search_paths()
   else
     local root = util.list_workspace_folders()[1]
-    -- print-paths emits a colon-separated list of .lean paths on the second line
+    if not root then root = vim.fn.getcwd() end
```

Comment:
This fixes a small side issue which is this now works even if you don't have a lean buffer open, it defaults then to the obvious cwd.